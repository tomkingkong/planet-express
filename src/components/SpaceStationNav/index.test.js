import React from 'react';
import { shallow } from 'enzyme';

import { SpaceStationNav } from '.';

describe('SpaceStationNav component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SpaceStationNav />);
    expect(wrapper).toMatchSnapshot();
  });
});