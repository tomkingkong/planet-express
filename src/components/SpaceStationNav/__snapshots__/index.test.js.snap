// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpaceStationNav component should match snapshot 1`] = `
<div
  className="SpaceStation"
>
  <div
    className="station__img"
    style={
      Object {
        "backgroundImage": "url(/images/station/ISS_overEarth_1.jpg)",
      }
    }
  />
  <h4
    className="station__title"
  >
    Welcome to the International Space Station
  </h4>
  <Link
    className="station__dest"
    replace={false}
    to="/ISS/destinations"
  >
    take a trip
  </Link>
  <Link
    className="station__lab"
    replace={false}
    to="/ISS/store"
  >
    visit spacelab
  </Link>
</div>
`;
